mmv1/templates/terraform/custom_flatten/memorystore_instance_automated_backup_config.go.tmpl

```diff
@@ -0,0 +1,81 @@
+{{/*
+	The license inside this block applies to this file
+	Copyright 2025 Google Inc.
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/ -}}
+func flattenMemorystoreInstanceAutomatedBackupConfig(v interface{}, d *schema.ResourceData, config *transport_tpg.Config) interface{} {
+	if v == nil {
+		return nil
+	}
+	original := v.(map[string]interface{})
+	if len(original) == 0 {
+		return nil
+	}
+	transformed := make(map[string]interface{})
+    // if automated_backup_config is not defined 
+	if len(d.Get("automated_backup_config").([]interface{})) < 1 {
```

Comment:
Drive-by comment: I think we need the same fix as https://github.com/GoogleCloudPlatform/magic-modules/pull/13401#discussion_r2017742196 here, so that we import correctly